frontend/src/components/housing/submission/SubmissionsNavigator.vue

```diff
@@ -218,10 +225,25 @@ watch(activeTabIndex, (newIndex) => {
       }
     });
   }
+
+  // Show toggle only for submissions and enquiries tab
+  showToggle.value = newIndex === TAB_INDEX.SUBMISSION || newIndex === TAB_INDEX.ENQUIRY ? true : false;
```

Comment:
Optional: This could be a computed. 

Ternary is not required. `newIndex === TAB_INDEX.SUBMISSION || newIndex === TAB_INDEX.ENQUIRY` will evaluate on its own.

Reply:
done